react/features/large-video/actions.web.ts

```diff
@@ -54,11 +52,11 @@ export function captureLargeVideoScreenshot() {
         canvasElement.style.display = 'none';
         canvasElement.height = parseInt(height, 10);
         canvasElement.width = parseInt(width, 10);
-        ctx.drawImage(videoElement, 0, 0);
+        ctx?.drawImage(videoElement, 0, 0);
```

Comment:
I don't think ctx can be undefined, we create it just above.

Reply:
`canvas.getContext` can return null. So this is possibly `null`